content/docs/user-guide/interactivity/user-interface/canvases/changing-size.md

```diff
@@ -9,8 +11,8 @@ Change your game UI canvas size to visualize how your canvas might look on other
 
 1. On the toolbar, click the arrow beside the resolution to see a list of commonly used canvas sizes for various platforms.
 
-![Canvas size dropdown menu on the toolbar](/images/user-guide/game_ui_editor/ui-canvas-change-size.png)
+![Canvas size dropdown menu on the toolbar](/images/user-guide/interactivity/user-interface/canvases/ui-canvas-change-size.png)
```

Comment:
The ordered list numbering resets after this image. To fix this, you'll need to indent each image in the list.